docs/tools/control.rst

```diff
@@ -12,13 +12,15 @@ Basic use
 version
 --------
 
-Print the current firmware version and vehicle platform of the attached CAN
+Print the current firmware version and vehicle description of the attached CAN
 translator:
 
 .. code-block:: bash
 
     $ openxc-control version
 
+An example response may be ``v7.2.1 (default)``, where the ``(default)`` vehicle description tag can be modified in the JSON file. See the ``passthrough`` example in [Low-level CAN Features](http://vi-firmware.openxcplatform.com/en/master/advanced/lowlevel.html).
```

Comment:
This link isn't rendering correctly. Also, this line of code is 3 lines long.  http://python.openxcplatform.com/en/document-revisions/tools/control.html